### DISCUSSION TITLE
Laravel Nova CSV Import

### DISCUSSION THREAD

**Discussion by simonhamp** [HassanZahirnia/laravel-package-ocean#173] [Package Suggestions]
https://github.com/simonhamp/laravel-nova-csv-import

**Comment by HassanZahirnia** [comment]
Hey Simon! Thanks for submitting your package! 😊

Unfortunately, I have not yet decided to add any packages from any of the Laravel sub-ecosystem. So things like Nova packages, Filament plugins, Livewire related packages, etc... Although you may find 1 or 2 here and there, but that's about it.

I wanna keep this project clean and simple, and only geared toward "directly" Laravel related packages.
